packages/documentation/src/content/docs/integration/requirements/idp.mdx

```diff
@@ -12,179 +12,130 @@ import {
 
 An identity provider (IdP) is a system or service that stores and manages user identity information, authentication, and consent. Examples of IdPs include OpenID Connect and Okta.
 
-Integration with an IdP is required if you plan to support outgoing payments via <LinkOut href="https://openpayments.dev">Open Payments</LinkOut>. The Open Payments standard requires interactive outgoing payment _grant_ requests, which precede outgoing payment requests. In an interactive grant request, explicit interaction by an individual (e.g., the client's end-user) is required to approve the grant. An example of an interaction is an end-user tapping _Approve_ in an app to authorize a payment.
+Open Payments requires any authorization server that issues interactive grants be integrated with an IdP. Interactive grants are used to gather consent. More information about interactive grants are available [below](#interactive-grants).
 
-Your IdP will:
-
-- Authenticate requests from clients, such as mobile apps, to create quotes and payments on Rafiki's backend
-- Facilitate interactions with the client's end-user to gather consent
+Responsibilities of your IdP include:
+- Providing an interface to gather end-user consent for a particular action
+- Sending the interaction choice (approve or deny) to your authorization server
+- Sending a request to your authorization server to finish the interaction
+- Redirecting the user after the interaction is complete
 
 :::note
-We provide Ory Kratos, a cloud-based user management system, for the identity and user management of your [Rafiki Admin](/admin/admin-user-guide) users. Kratos is for internal use only and **cannot** be used as your client-facing IdP.
+We provide Ory Kratos, a cloud-based user management system, for the identity and user management of your Rafiki Admin users. Kratos is for internal use only and **cannot** be used as your IdP for Open Payments.
 :::
 
-### Interactions and consent
-
-Before an outgoing payment is created via Open Payments, an outgoing payment _grant_ must be issued.
+## Interactive grants
 
-Outgoing payment grant requests must be interactive. This means the request requires explicit interaction, often from the a client's end-user, to gather consent (permission) before creating the outgoing payment. The interaction is facilitated by your IdP.
+In Open Payments, grants are used to indicate a resource owner, such as an account holder, has given a piece of software, such as a mobile app, permission (consent) to act on their behalf. 
 
-Your IdP:
+Open Payments requires that consent is collected via an interactive grant before an <LinkOut href="https://openpayments.dev/introduction/op-concepts/#outgoing-payment">outgoing payments</LinkOut> request is issued. A grant is interactive when explicit interaction by an entity (e.g., the software's end user) is required to approve or deny the grant. Tapping an *Approve* button to authorize a payment is an example of an explicit interaction.
```

Comment:
A small, but potentially useful distinction:

> Open Payments requires that consent is collected via an interactive grant before an outgoing payments request is issued

The Open Payments API itself doesn't mandate any grants to be interactive per se (it just says a grant request request can be non-interactive or interactive). The opinionated Open Payments AS in Rafiki _does_, however.